docs/Developers/Debug_Bears.rst

```diff
@@ -0,0 +1,78 @@
+Debug Bears
+===========
+
+This document provide a brief overview of coala's Debugging Interface.
+The Debugging Interface will help users to debug Bear code and step through
+it using the pdb interface.
+
+Details
+-------
+
+After invoking coala's debugger it will step into the ``run()`` method of
+bear regardless of using ``yield`` or not and will step out as soon as bear
+finish analysis.
+
+.. note::
+
+    Pdb's quit-command (``q``) has been remapped so coala continues its
+    normal execution without aborting. So, ``quit`` or `q` will first clear
+    all the breakspoints then continue ( ``c`` or ``continue`` ) execution.
```

Comment:
--> break~~s~~points
--> all ~~the~~ breakpoints ~~and~~ then continue execution
I would also remove the "(c or continue)" part, you are describing the `q` command, and suddenly you are mentioning the `c` command :)